master/internal/api_search.go

```diff
@@ -169,3 +169,230 @@ func (a *apiServer) DeleteSearchTag(
 	}
 	return &res, nil
 }
+
+func BulkFiltersSearchtToExperiment(filter *apiv2.BulkSearchFilters) *apiv1.BulkExperimentFilters {
```

Comment:
Small typo here -- hopefully a tool can help rename it (F2 in VSCode, or find+replace would probably do it in this case)

Reply:
Fixed